lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -5848,4 +5905,10 @@ is the only supported options.</xs:documentation>
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="MacroImportType">
+    <xs:restriction base="xs:anyURI">
+      <xs:pattern value="([a-zA-Z0-9_\-/\.])+.xml"/>
```

Comment:
Actually I think this is better...
```suggestion
      <xs:pattern value="[a-zA-Z0-9_\-\.]+.xml"/>
```

Reply:
I am assuming the macro files will be in the same directory as the tool importing them. Is that correct?